### PR TITLE
Easy integrate bundle

### LINKED ISSUE #872: Easy-to-integrate MONAI bundle
**Is your feature request related to a problem? Please describe.**
Now users still find it too challenging to get started with the monai bundle and are unaware of its capabilities. Users don't know how to use the stored weights of a model, training, inference, post-processing transform sequences and other information included in bundle. To demonstrate users how to quickly integrate the bundle into their own procedures, we need a simple application.

**Describe the solution you'd like**
We can create a straightforward ensemble or TTA(Test time augmentation) application and instruct the users on how to use the existing bundle.

### PR DESCRIPTION
**Pull request by KumoLiu**
Fixes Project-MONAI/MONAI#4960 .

### Description
Create a straightforward ensemble application to instruct users on how to use the existing bundle.

### Status
**Work in progress**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by Nic-Ma** [comment]
Hi @KumoLiu ,

Thanks for the quick implementation.
As you developed the tutorial in python program, could you please help add a README file in the `tutorials/model_zoo/` to describe your tutorial? We will add more to this file when more tutorials are ready.

Thanks.

File: model_zoo/easy_integrate_bundle.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+                print(line, flush=True)
+
+        logger.info(f"Return code: {process.returncode}")
+        process.stdout.close()
+
+
+if __name__ == '__main__':
+    """
+    Usage:
+        python easy_integrate_bundle.py --bundle_root /workspace/Code/Bundles/spleen_ct_segmentation --dataset_dir /workspace/Data/Task09_Spleen
```
Comment (wyli) [inline model_zoo/easy_integrate_bundle.py:222]: would be great to describe 
- the goal of this script
- the expected output
- the steps to achieve the goal (in more detail): 1. dataset splitting, 2. training multiple models using each split, 3, compute ensemble results using the models
- assumption of this script (how much GPU memory? which model zoo item? structure of the data list and dataset folders?)

File: model_zoo/easy_integrate_bundle.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+        if ensemble == 'Mean':
+            post_tranform["transforms"].insert(0, ensemble_tranform)
+        elif ensemble == 'Vote':
+            post_tranform["transforms"].insert(-1, ensemble_tranform)
+        else:
+            raise NotImplementedError
+        bundle_inference_config.update({Const.KEY_INFERENCE_POSTPROCESSING: post_tranform})
+
+        # update network weights
+        _networks = [bundle_inference_config.get_parsed_content("network")]*5
```
Comment (Nic-Ma) [inline model_zoo/easy_integrate_bundle.py:107]: missed blank before and after `*`.

Thanks.

**Review by Nic-Ma — APPROVED** [review]
Thanks for the update.
Looks good to me now.
@wyli Do you have any other comments?

File: model_zoo/ README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+# Easy Integrate Bundle Tutorial
```
Comment (wyli) [inline model_zoo/ README.md:1]: This file name contains a typo of white space before 'readme'

**Pull request merged**